cmd/lifecycle/builder.go

```
@@ -36,6 +37,11 @@ func (b *buildCmd) Args(nargs int, args []string) error {
 	return nil
 }
 
+func (b *buildCmd) DropPrivileges() error {
+	// builder should never be run with privileges
```

Maybe return an error if UID != 0?